src/analyzing/AST_to_IL.ml

```diff
@@ -270,6 +270,22 @@ let def_expr_evaluates_to_value (lang : Lang.t) =
   | Elixir -> true
   | _else_ -> false
 
+let is_constructor env ret_ty id_info =
+  match id_info.G.id_resolved.contents with
+  | Some (G.GlobalName (ls, _), _) -> (
+      (* TODO this condition is a little too loose, but should work in
+         practice could we tighten this up? *)
+      env.lang =*= Lang.Python
+      && List.length ls >= 3 (* Module + Class + __init__ *)
+      && (match List_.last_opt ls with
+         | Some "__init__" -> true
+         | _ -> false)
+      &&
+      match ret_ty with
+      | { G.t = G.TyN _; _ } -> true
```

Comment:
Add a comment to clarify what this means, I'm guessing, the result type is a "type name", presumably the name of a class?